test/plugin/test_out_redis.rb

```diff
@@ -21,6 +21,7 @@ def test_configure
     assert_equal 'localhost', @d.instance.host
     assert_equal 6379, @d.instance.port
     assert_equal 1, @d.instance.db_number
+    assert_nil @d.instance.password
```

Comment:
I've added `test_configure_with_password` test case for checking `password` parameter.
How about this?